.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "pd-lib-builder"]
```

Comment:
i'd rather use `git subtree` than `git submodule`. it makes it much easier to build from a "release tarball" (as opposed to to "git clone")

Reply:
Ok.

Reply:
I may just include the lib builder directly as with earplug~. Easy enough for just 1 file.

Reply:
`git subtree` makes it easier to upgrade: keeping history *and* not having to worry about whether the it stays single-file.

anyhow: just make sure to use `PDLIBBUILDER_DIR`

Reply:
Yup and we should update the pdlibbuilder hello-world example to use it as well. That's what I copy/pasted yesterday...

Reply:
you mean like https://github.com/pure-data/helloworld/commit/dd1554035fcf9992a4c887853a682c7c13d5454c? (note the date)

Reply:
ah, by *it* did you mean `git subtree` (rather than `PDLIBBUILDER_DIR`, as i assumed)?

Reply:
No, I guess I used on old cloned copy of hell world.

Reply:
In lookin at this again, I already set it up as a subtree and now I remember that I find subtrees confusing. In this case, it's a single file to update and I don't want to pull in all the lib builder tests with it.